aquarius/ddo_checker/shacl_schemas/v4/remote_4.0.0.ttl

```diff
@@ -241,7 +291,12 @@ schema:ServiceShape
     ] ;
     sh:property [
         sh:path schema:additionalInformation ;
-    ] ; .
+    ] ;
+    sh:property [
+        sh:path schema:consumerParameters;
+        sh:node schema:ConsumerParametersShape ;
+        sh:maxCount 1;
```

Comment:
Same as previous comment.  Shouldn't this line be removed, so that there can be more than 1 `ConsumerParameter`?